### PR TITLE
Data migration issue1329

### LINKED ISSUE #1329: Data Model - Migration script from master to Draw.io
## Tasks
- [ ] Implement a migration from master to draw.io data model

### PR DESCRIPTION
**Pull request by Mussuranae**
Migration script from master to draw.io.
We will need to update the firestore.ts and the call to some data.
Fixes #1329

### CODE REVIEW

File: apps/backend-ops/src/firestoreMigrations/0003.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+  const newInvitationDoc = invitations.docs.map( async (invitDocSnapshot: any): Promise<any> => {
+    const invitationData = invitDocSnapshot.data();
+    const {organization, user} = invitationData;
+
+    delete invitationData.organization;
+    delete invitationData.user;
+
+    const newData = {
+      ...invitationData,
```
Comment (phpgeekfr) [inline apps/backend-ops/src/firestoreMigrations/0003.ts]: don't forget to update draw.io

File: apps/backend-ops/src/firestoreMigrations/0003.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  const newOrgData = organizations.docs.map(async (orgDocSnapshot: any): Promise<any> => {
+    const organizationData = orgDocSnapshot.data();
+    const { name, movieIds } = organizationData;
+
+    delete organizationData.name;
+
+    const newData = {
+      ...organizationData,
+      cart: [],
+      denomination: {
```
Comment (phpgeekfr) [inline apps/backend-ops/src/firestoreMigrations/0003.ts]: denomination: not implement yet in angular code (quite a lot of work to do on this)

File: apps/backend-ops/src/firestoreMigrations/0003.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+  const newOrgData = organizations.docs.map(async (orgDocSnapshot: any): Promise<any> => {
+    const organizationData = orgDocSnapshot.data();
+    const { name, movieIds } = organizationData;
+
+    delete organizationData.name;
+
+    const newData = {
+      ...organizationData,
+      cart: [],
```
Comment (phpgeekfr) [inline apps/backend-ops/src/firestoreMigrations/0003.ts]: cart[] is now a subcollection, see draw.io

File: apps/backend-ops/src/firestoreMigrations/0003.ts [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      denomination: {
+        full: name,
+        public: name
+      },
+      movieIds: {
+        archipelContent: movieIds
```
Comment (phpgeekfr) [inline apps/backend-ops/src/firestoreMigrations/0003.ts]: not implement yet, @see #618

File: apps/backend-ops/src/firestoreMigrations/0003.ts (hunk 1 above)
Comment (Mussuranae) [reply apps/backend-ops/src/firestoreMigrations/0003.ts]: I was waiting to talk with you about these changes before update the draw, but now it's ok :)

File: apps/backend-ops/src/firestoreMigrations/0003.ts (hunk 2 above)
Comment (Mussuranae) [reply apps/backend-ops/src/firestoreMigrations/0003.ts]: A lot of these changes are not implemented yet in the code, this is why only one or two people have to do it and update the code (and the firestore.ts files) before everyone run this script.

File: apps/backend-ops/src/firestoreMigrations/0003.ts (hunk 3 above)
Comment (Mussuranae) [reply apps/backend-ops/src/firestoreMigrations/0003.ts]: I will update that

**Pull request closed without merging**